aea_projection.py

```diff
@@ -337,6 +338,28 @@ def set_ylim(self, *args, **kwargs):
         Axes.set_ylim(self, *args, **kwargs)
         self._update_affine()
 
+    def histmap(self, ra, dec, nside=32, weights=None, mean=False, **kwargs):
+        r = histogrammap(ra, dec, nside, weights)
+
+        if weights is not None:
+            w, N = r
+        else:
+            w = r
+        if mean:
+            mask = N != 0
+            w[mask] /= N[mask]
+        else:
+            mask = w > 0
+
+        return w, mask, self.mapshow(w, mask, nest=False, **kwargs)
+
+    def mapshow(self, map, mask, nest=False, **kwargs):
```

Comment:
this method does not seem to be using `map` at all !? I might be missing something, but that doesn't look right